Integrations/SignalSciences/SignalSciences.py

```diff
@@ -0,0 +1,1424 @@
+LIST_OF_SITE_LISTS_TITLE = 'Signal Sciences - list of site lists'
+LIST_OF_CORP_LISTS_TITLE = 'Signal Sciences - list of corp lists'
+LIST_OF_EVENTS_TITLE = 'Signal Sciences - list of events'
+LIST_OF_REQUESTS_TITLE = 'Signal Sciences - list of requests'
+CREATE_SITE_LIST_TITLE = "Signal Sciences - creating a new site list \n\n List {0} has been successfully created"
+CREATE_CORP_LIST_TITLE = "Signal Sciences - creating a new corp list \n\n List {0} has been successfully created"
+DELETE_CORP_LIST_TITLE = "### Signal Sciences - deleting corp list \n\n List {0} has been successfully removed"
+EXPIRE_EVENT_TITLE = "### Signal Sciences - expiring event \n\n Event {0} has been successfully expired"
+WHITELIST_REMOVE_IP_TITLE = '### Signal Sciences - Removing an IP from Whitelist \n\n ' \
+                            'The IP {0} has been successfully removed from Whitelist.'
+DELETE_SITE_LIST_TITLE = "### Signal Sciences - deleting site list \n\n The list has been succesfully removed"
+BLACKLIST_REMOVE_IP_TITLE = '### Signal Sciences - Removing an IP from Blacklist \n\n ' \
+                            'The IP {0} has been successfully removed from Blacklist.'
+IP_ADDED_TO_WHITELIST_TITLE = "The IP {0} has been successfully added to whitelist."
+IP_ADDED_TO_BLACKLIST_TITLE = "The IP {0} has been successfully added to blacklist."
+
+
+'''TABLE HEADERS'''
+ADD_IP_HEADERS = ['Source', 'Note', 'Expiration date']
+WHITELIST_OR_BLACKLIST_HEADERS = ['ID', 'Source', 'Expiry Date', 'Note', 'Created Date', 'Created By']
+LIST_HEADERS = ['Name', 'ID', 'Type', 'Entries', 'Description', 'Created By', 'Created Date', 'Updated Date']
+GET_SITE_HEADERS = ['Name', 'Created Date']
+EVENT_HEADERS = ['ID', 'Timestamp', 'Source', 'Remote Country Code', 'Action', 'Reasons', 'Remote Hostname',
+                 'User Agents', 'Request Count', 'Tag Count', 'Window', 'Date Expires', 'Expired By']
+REQUEST_HEADER = ['ID', 'Timestamp', 'Remote Country Code', 'Remote Hostname', 'Remote IP', 'User Agent',
+                  'Method', 'Server Name', 'Protocol', 'Path', 'URI', 'Response Code', 'Response Size',
+                  'Response Millis', 'Agent Response Code', 'Tags']
+ALERT_HEADERS = ['ID', 'Site ID', 'Created Date', 'Tag Name', 'Action', 'Long Name', 'Interval (In Minutes)',
+                 'Threshold', 'Block Duration Seconds', 'Skip Notifications', 'Enabled']
+
+
+'''List Types dict'''
+
+LEGAL_SIGSCI_LIST_TYPES = {
+    'ip',
+    'country',
+    'string',
+    'wildcard'
+}
+
+
+''' HELPER FUNCTIONS '''
+
+
+def camel_case_to_spaces(string_in_camel_case):
+    """Given a string in camelcase, will turn it into spaces
+
+    Args:
+        string_in_camel_case(String): the string in camel case
+
+    Returns:
+        A new string, separated by spaces and every word starts with a capital letter
+    """
+    string_with_underscores = camel_case_to_underscore(string_in_camel_case)
+    words_in_string = string_with_underscores.split('_')
```

Comment:
```suggestion
    words_in_string = string_with_underscores.replace(' ')
```
That's all you need

Reply:
It seems like 'replace' takes two arguments, are you sure?

Reply:
I meant replace _ with a space

Reply:
Done deal